rust/src/arrow.rs

```diff
@@ -201,11 +201,11 @@ pub trait FixedSizeListArrayExt {
     ///         Some(vec![Some(8), Some(9)])
     /// ], 2))
     /// ```
-    fn try_new<T: Array>(values: T, list_size: i32) -> Result<FixedSizeListArray>;
+    fn try_new_from_values<T: Array>(values: T, list_size: i32) -> Result<FixedSizeListArray>;
```

Comment:
Arrow added their own `try_new` methods, so we have to rename these.

Reply:
The functionality is the same as [`FixedSizeListArray::try_new()`](https://docs.rs/arrow-array/42.0.0/arrow_array/array/struct.FixedSizeListArray.html#method.try_new) ? Should we just deprecate this method?

Reply:
Maybe? Or leave it as a wrapper? The new method takes 4 arguments, which is a little annoying to type out. The caller would have to create the `Field` with the correct matching type.